accessibility-checker-engine/help/HAAC_Aria_SvgAlt.mdx

```diff
@@ -40,8 +40,8 @@ Example:
 
 ### About this requirement
 
-[IBM 4.1.2 Name, Role, Value](https://www.ibm.com/able/requirements/requirements/#4_1_2)
-[WAI-ARIA Graphics Module](https://www.w3.org/TR/graphics-aria-1.0/)
+* [IBM 4.1.2 Name, Role, Value](https://www.ibm.com/able/requirements/requirements/#4_1_2)
+* [ARIA Graphics Module](https://w3c.github.io/graphics-aria/)
```

Comment:
This is spec and not ARIA practices. Shouldn't this be the latest working draft rather than the editors draft?

Reply:
No, becasue the last published version is from 2018, while this is July 2021 that has been updated for ARIA 1.1.  When ARIA 1.2 goes final (its candidate now), this draft should be published.

Reply:
Should probably be https://www.w3.org/TR/graphics-aria/. The ARIA Graphics module isn't part of core ARIA and doesn't have the same spec, nor support timelines. The 2018 published spec says it's already updated for 1.1.

Reply:
OK, reverting back to published for now. Will open a new issue if working group editors feel the latest 2021 draft is better.

https://www.w3.org/TR/graphics-aria/

Reply:
OK, changed it back and pushed change in branch